app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
+
+  def learn_more
+  end
+
+  def tools_resources
+  end
+
+  def locations
+  end
+end
```

Comment:
Added in a Static Pages Controller. Although we are not passing in anything to the views at this point. We will be pulling the content from the database we went ahead and included these now for visibility.